src/app/features/wallets/wallets.page.html

```diff
@@ -32,12 +32,19 @@ <h3 class="num-text">NUM</h3>
       </ion-row>
       <ion-row id="mainnet-points-row" class="text-black">
         MAINNET:
-        <span
-          class="mainnet-points"
+        <ng-container
           *ngIf="!(isLoadingBalance$ | ngrxPush); else balanceLoading"
         >
-          ${{ mainNumBalance$ | ngrxPush | number : '1.2-2' }}
-        </span>
+          <span class="mainnet-points">
+            ${{ mainNumBalance$ | ngrxPush | number : '1.2-2' }}
+          </span>
+          <ion-icon
+            name="information-circle"
+            #mainnetBalanceTooltip="matTooltip"
+            (click)="mainnetBalanceTooltip.toggle()"
+            [matTooltip]="'Mainnet NUM balance'"
```

Comment:
@bafu, once Sofia provides text and translations I will change L45. But for now this PR looks like [this](https://i.imgur.com/EWuWEFM.mp4)